lib/resources/etc_fstab.rb

```diff
@@ -0,0 +1,119 @@
+# encoding: utf-8
+# copyright:
+# author: Matthew Dromazos
+
+require 'utils/parser'
+require 'pry'
+class EtcFstab < Inspec.resource(1)
+  name 'etc_fstab'
+  desc 'Use the etc_fstab InSpec audit resource to check the configuration of the etc/fstab file.'
+  example "
+  removable_media = etc_fstab.removable_media_file_systems
+  removable_media.each do |media|
+    describe media do
+      its ( 'mount_options' ) { should include 'nosuid' }
+    end
+  end
+
+  nfs_systems = etc_fstab.nfs_file_systems
+  if removable_media != []
```

Comment:
I'm not sure why this is part of this example. Can we make this a bit easier for our users to understand by removing this condition?